ic-os/defs.bzl

```diff
@@ -553,17 +553,19 @@ EOF
         ],
         outs = ["launch_local_vm_script"],
         cmd = """
-        IMAGE="$(location :disk-img.tar)"
+        IMAGE="$$PWD/$(location :disk-img.tar)"
         cat <<EOF > $@
 #!/usr/bin/env bash
 set -euo pipefail
 cd "\\$$BUILD_WORKSPACE_DIRECTORY"
-TEMP=\\$$(mktemp -d)
+TEMP=\\$$(mktemp -d --suffix=.qemu-launch-remote-vm)
+# Clean up after ourselves when exiting.
+trap 'rm -rf \\$$TEMP' EXIT
 CID=\\$$((\\$$RANDOM + 3))
-cp $$IMAGE \\$$TEMP
 cd \\$$TEMP
-tar xf disk-img.tar
-qemu-system-x86_64 -machine type=q35,accel=kvm -enable-kvm -nographic -m 4G -bios /usr/share/ovmf/OVMF.fd -device vhost-vsock-pci,guest-cid=\\$$CID -drive file=disk.img,format=raw,if=virtio -netdev user,id=user.0,hostfwd=tcp::2222-:22 -device virtio-net,netdev=user.0
+tar xSf $$IMAGE
```

Comment:
I don't think GNU tar that we use in devenvs/build container supports sparse extraction.

> This option is meaningful only when creating or updating archives. It has no effect on extraction.

https://www.gnu.org/software/tar/manual/html_node/sparse.html

Reply:
I think that information is wrong.

```
root@SetupOS:~# cd /tmp
root@SetupOS:/tmp# truncate -s 1G filetotar
root@SetupOS:/tmp# du -hs filetotar
0       filetotar
root@SetupOS:/tmp# tar cvSzf filetotar.tar.gz filetotar
filetotar
root@SetupOS:/tmp# mkdir extracted
root@SetupOS:/tmp# cd extracted
root@SetupOS:/tmp/extracted# tar zxSvmf ../filetotar.tar.gz
filetotar
root@SetupOS:/tmp/extracted# du -hs filetotar
0       filetotar
root@SetupOS:/tmp/extracted# ls -la filetotar
-rw-r--r--. 1 root root 1073741824 Oct 16 12:41 filetotar
root@SetupOS:/tmp/extracted# 
```

Zero bytes extracted, 1G file successfully extracted.